src/execution_plan/execution_plan.h

```diff
@@ -28,12 +28,6 @@ struct ExecutionPlan {
 /* Creates a new execution plan from AST */
 ExecutionPlan *NewExecutionPlan(void);
 
-// Sets an AST segment in the execution plan.
-void ExecutionPlan_SetAST(ExecutionPlan *plan, AST *ast);
-
-// Gets the AST segment from the execution plan.
-AST *ExecutionPlan_GetAST(const ExecutionPlan *plan);
-
```

Comment:
we don't needs these two functions any longer?

Reply:
No, they are completely unused.